src/console_reporter.cc

```diff
@@ -53,7 +53,7 @@ bool ConsoleReporter::ReportContext(const Context& context) {
 }
 
 void ConsoleReporter::PrintHeader(const Run& run) {
-  std::string str = FormatString("%-*s %13s %13s %10s", static_cast<int>(name_field_width_),
+  std::string str = FormatString("%-*s %13s %15s %12s", static_cast<int>(name_field_width_),
```

Comment:
I'm not sure, is this still needed now that no alignment happens?

Reply:
Yes. There was a bug originally where we misaligned the last couple fields of the header when complexity was involved.